src/pages/index.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { FC, useEffect, useRef } from 'react'
 import Vizzu, { AnimTarget } from 'vizzu'
+import VizzuModule from './../../node_modules/vizzu/dist/cvizzu.wasm?url'
```

Comment:
I cloned this, and changed this path to `import VizzuModule from 'vizzu/dist/cvizzu.wasm?url'`. Looks like the relative pathing isn't needed here.